src/dicarlo_lab_to_nwb/conversion/convert_session.py

```diff
@@ -75,6 +75,9 @@ def convert_session_to_nwb(
     output_dir_path.mkdir(parents=True, exist_ok=True)
 
     # session_id defines the name of the NWB file
+    # Break this down so recording_id is separated int its components, consideer using a camel case veersion
```

Comment:
I left a comment here on where you have to do the changes.|

For reference, this is the place where the info is used in aggregation:
https://github.com/catalystneuro/dicarlo-lab-to-nwb/blob/a376254e97a498df9e0fbe18e49a7c66e1e2a20f/src/dicarlo_lab_to_nwb/conversion/aggregation.py#L313-L318

Reply:
Fixed splitting the recording_id into stimulus_name (camel case), date, and time. 
Also modified lines 314-315 in aggregation.py to follow the session_id convention written here in line 73

Reply:
Thanks for doing the second :+1:

Reply:
Will push updates to yoon_updated after testing